spec/controllers/api/api_controller_spec.rb

```diff
@@ -28,7 +29,7 @@ class DummyController < Api::ApiController
       let(:bearer_token) { "Bearer #{token}" }
 
       it "requests authentication via the http header" do
-        expect(response.status).to eq(200)
+        expect(response.status).to eq(204)
```

Comment:
Not entirely sure why this changed from `200` to `204` 🤔  Is it just the default for the empty `#index` above?